git-push-fork-to-upstream-branch

```diff
@@ -15,10 +15,10 @@ fi
 
 SOURCE_GH_USER=$(echo "$BRANCH_SPEC" | awk -F: '{print $1}')
 SOURCE_BRANCH=$(echo "$BRANCH_SPEC" | awk -F: '{print $2}')
-REPO_NAME=$(basename $(git rev-parse --show-toplevel))
+REPO_NAME=$(git remote get-url --push origin | cut -d '/' -f 2 | cut -d '.' -f 1)
```

Comment:
This assumes that there's a remote called `origin`, yes? That's _probably_ always the case? I suppose we could simply document that as an assumption.